src/Microsoft.ML.FastTree/GamClassification.cs

```diff
@@ -173,8 +175,10 @@ private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "GAM BINP",
-                verWrittenCur: 0x00010001,
-                verReadableCur: 0x00010001,
+                // verWrittenCur: 0x00010001, // Initial
+                // verWrittenCur: 0x00010001, // Added Intercept but collided from release 0.6-0.9
+                verWrittenCur: 0x00020001,    // Added Intercept (version revved to address collisions)
```

Comment:
>0x00020001 [](start = 31, length = 10)

I think in general we upgrade version by last digit, i.e. `0x0010002` instead of `0x00020001`

Reply:
Done.